docs/modules/ROOT/pages/index.adoc

```diff
@@ -53,7 +53,7 @@ cargo build --release
 ./target/release/parachain-template-node build-spec --chain plain-parachain-chainspec.json --disable-default-bootnode --raw > raw-parachain-chainspec.json
 ```
 
-* Run two nodes and wait till it syncs with the Rococo relay chain (it may take up to two days to sync it):
+* Run two nodes and wait until it syncs with the Rococo relay chain. This can take a fairly long time(up to 2 days), so we can use the `fast-unsafe` flag to make the process faster since we are on a testnet(~ 3 hours). `fast` downloads the blocks without executing the transactions, and `unsafe` skips downloading the state proofs(which we are ok with since it is Rococo).
 +
 ```bash
 ./target/release/parachain-template-node \
```

Comment:
Not directly related to the issue. But when updating the docs I had a couple questions that I didn't know what they are and I don't think we explain it here.

Is there a way we can automatically fill these for users? And if not make it easier for newcomers to understand where they can get this?
--base-path <path to datadir>
--chain <path to the Rococo chainspec>

Reply:
- `--base-path` -> this is where the downloaded chain state will be stored, it can be any folder
- `chain` -> <path to the Rococo chainspec>, they can get this from `polkadot-sdk` repository from this directory: `/polkadot/node/service/chain-specs/` and here is the link: https://github.com/paritytech/polkadot-sdk/tree/release-polkadot-v1.10.0/polkadot/node/service/chain-specs

Reply:
Thanks. I added those two references to the PR. Happy to work with you in the other changes for the docs :)